### DISCUSSION TITLE
Understand major version between two versions

### DISCUSSION THREAD

**Discussion by Tristan-gu** [invertase/melos#808] [Q&A]
I don't understand something about major versions.

I have 2 packages
- packages
    |- my_package_1 : 1.0.0
    |- my_package_2 : 1.0.0

I make some commits:
- packages
    |- my_package_1 : feat! commit
    |- my_package_2 : fix: commit

And make a beta version with preid argument, the major version is good on 'my_package_1'
- packages
    |- my_package_1 : 2.0.0.beta.0
    |- my_package_2 : 1.0.1.beta.0

If I make a breaking change commit on 'my_package_2' and make a new preid version.
- packages
    |- my_package_1 : 2.0.0.beta.1
    |- my_package_2 : 1.0.1.beta.1

the my_package_2 is not updating the major version, is this normal ?

**Comment by spydon** [comment]
That sounds like a bug, can you file it as an issue instead of a discussion?

**Reply by Tristan-gu** [reply]
Done here https://github.com/invertase/melos/issues/809
